acceptance/redis32/main.go

```diff
@@ -55,34 +84,72 @@ func handler(w http.ResponseWriter, r *http.Request) {
 		writeError(w, err)
 		return
 	}
-	client.Close()
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
 }
 
-func newPool(addr string, password string) *redis.Pool {
-	return &redis.Pool{
-		MaxIdle:     3,
-		MaxActive:   10,
-		IdleTimeout: 2 * time.Second,
-		Dial: func() (redis.Conn, error) {
-			c, err := redis.Dial("tcp", addr)
-			if err != nil {
-				return nil, err
-			}
-			if _, err := c.Do("AUTH", password); err != nil {
-				c.Close()
-				return nil, err
-			}
-			return c, nil
-		},
-		TestOnBorrow: func(c redis.Conn, t time.Time) error {
-			threshold := 5 * time.Second
-			if time.Since(t) < threshold {
-				return nil
-			}
-			_, err := c.Do("PING")
-			return err
-		},
+func info(w http.ResponseWriter, r *http.Request) {
+	client = pool.Get()
+	defer client.Close()
+
+	parameter := r.URL.Query().Get("s")
+
+	if parameter == "" {
+		parameter = "all"
 	}
+
+	infoString, err := redis.String(client.Do("INFO", parameter))
+
+	if err != nil {
+		writeError(w, err)
+		return
+	}
+
+	infoMap := make(map[string]string)
+
+	for _, line := range strings.Split(infoString, "\r\n") {
+		part := strings.Split(line, ":")
+		if len(part) == 2 {
+			infoMap[string(part[0])] = string(part[1])
+		}
+	}
+
+	jresp, _ := json.Marshal(infoMap)
+
+	w.Header().Set("Content-Type", "application/json")
+	w.WriteHeader(http.StatusOK)
+	w.Write(jresp)
+
+}
+
+func configGet(w http.ResponseWriter, r *http.Request) {
+	client = pool.Get()
+	defer client.Close()
+
+	parameter := r.URL.Query().Get("p")
+
+	if parameter == "" {
+		parameter = "*"
+	}
+
+	primaryConfig, err := redis.StringMap(client.Do("CONFIG", "GET", parameter))
+
+	if err != nil {
+		writeError(w, err)
+		return
+	}
+
+	w.WriteHeader(http.StatusOK)
+
+	if parameter == "*" {
```

Comment:
yeah. I was going for one less `jq` call in the `redis-ha.sh` while also learning a bit more Go. I'm fine with reverting it back once the Shell script is in a better place.